bpaingest/projects/stemcells/ingest.py

```diff
@@ -490,19 +513,22 @@ def parse_spreadsheet(self, fname, additional_context, mode):
             field_spec = [
                 fld("pool_id", 'raw file name', coerce=files.proteomics_raw_extract_pool_id),
             ]
+        # allow for various unit symbols or none to be used (in this case: micrograms symbol)
+        units_regex = '[' + b'\xc2\xb5'.decode('utf-8') + ']?'
         field_spec += [
             fld("facility", 'facility'),
             fld("sample_fractionation", 'sample fractionation (none/number)'),
             fld("lc_column_type", 'lc/column type'),
             fld("gradient_time", re.compile(r'gradient time \(min\).*')),
-            fld("sample_on_column", 'sample on column (g)'),
+            fld("sample_on_column", re.compile(r'sample on column \(' + units_regex + re.escape('g)')), optional=True),
             fld("mass_spectrometer", 'mass spectrometer'),
```

Comment:
you only use `units_regex` in one place, why not just write a regex in place here? you can put in the literal unicode, python3 is fine with utf8 in files, and it would be clearer that the `.decode` call